core/admin/mailu/internal/views/postfix.py

```diff
@@ -145,8 +145,9 @@ def postfix_sender_login(sender):
     localpart = localpart[:next((i for i, ch in enumerate(localpart) if ch in flask.current_app.config.get('RECIPIENT_DELIMITER')), None)]
     destinations = models.Email.resolve_destination(localpart, domain_name, True) or []
     destinations.extend(wildcard_senders)
+    destinations.extend(i[0] for i in models.User.query.filter_by(allow_spoofing=True).with_entities(models.User.email).all())
     if destinations:
-        return flask.jsonify(",".join(idna_encode(destinations)))
+        return flask.jsonify(",".join(idna_encode(list(set(destinations)))))
     return flask.abort(404)
```

Comment:
You don't need to use `list()` here.
What about this for the function:
```
def postfix_sender_login(sender):
    wildcard_senders = {s for s in flask.current_app.config.get('WILDCARD_SENDERS', '').lower().replace(' ', '').split(',') if s}
    localpart, domain_name = models.Email.resolve_domain(sender)
    if localpart is None:
        return flask.jsonify(",".join(wildcard_senders)) if wildcard_senders else flask.abort(404)
    destinations = set(models.Email.resolve_destination(localpart, domain_name, True) or [])
    destinations.update(wildcard_senders)
    destinations.update(a[0] for a in models.User.query.filter_by(allow_spoofing=True).with_entities(models.User.email))
    if destinations:
        return flask.jsonify(",".join(idna_encode(destinations)))
```

Reply:
You're missing a line, but yes, sure. Done in 19bd9362